src/wwwroot/config.webpack/webpack.development.js

```diff
@@ -1,12 +1,17 @@
+var webpack = require('webpack');
+
 module.exports = function() {
    return {
       debug: true,
       devtool: 'source-map ',
       devServer: {
          historyApiFallback: true,
-         hot: true,
+         //hot: true,
```

Comment:
please, uncomment this props.
You already have ` new webpack.HotModuleReplacementPlugin()` into configs, so definitely now all works fine